astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx

```diff
@@ -43,10 +45,30 @@ export const NotificationsBell: VFC<NotificationsBellProps> = ({
   }
 
   return accountId && FEATURE_FLAGS.NOTIFICATIONS ? (
-    <Link href="/notifications" passHref>
-      <div className={cn(styles.root, className)} ref={rootRef}>
-        {renderBellIcon()}
-      </div>
-    </Link>
+    <GenericDropdown
```

Comment:
The dropdown was removed from this place by design, so we need a link only

Reply:
These changes are not from my PR. PR closed, new PR #252